source/reference/operator/aggregation/geoNear.txt

```diff
@@ -278,7 +278,9 @@ Minimum Distance
 
 The following example uses the option ``minDistance`` to specify
 the minimum distance from the center point that the documents can be.
-The following aggregation finds all documents that 
+The following aggregation finds all documents with a location at least
+2 meters from the center [ -73.99279 , 40.719296 ] and ``category`` 
+equal to ``Parks``.
```

Comment:
Hi Ian --

Tiny nit: let's render the coordinates ( `[ -73.99279 , 40.719296 ]` ) as monospace also as with the previous example intro.

Reply:
Hi Andrew, added monospace per request. Thanks!

Reply:
Thanks! Looks good!